lib/explorer/data_frame.ex

```diff
@@ -3922,28 +3922,7 @@ defmodule Explorer.DataFrame do
       end
     end
 
-    distinct_pivot_columns =
-      df[names_from]
-      |> Series.distinct()
-      |> Series.to_list()
-      |> Enum.map(&String.replace_prefix(&1, "", opts[:names_prefix]))
-
-    out_columns = id_columns ++ distinct_pivot_columns
-
-    new_dtypes =
-      for column <- distinct_pivot_columns, into: %{}, do: {column, df.dtypes[values_from]}
-
-    out_dtypes =
-      df.dtypes
-      |> Map.drop([names_from, values_from])
-      |> Map.merge(new_dtypes)
-
-    out_groups = df.groups -- [names_from, values_from]
-
-    out_df = %{df | dtypes: out_dtypes, names: out_columns, groups: out_groups}
-
     Shared.apply_impl(df, :pivot_wider, [
```

Comment:
Do we need to do something with groups? I assume this will always ungroup it now, right? But I think the previous version was wrong in regards to groups (it was keeping groups that no longer exist in the dataframe).

Reply:
Ignore me.